pkg/api/apiloader_test.go

```diff
@@ -105,6 +106,12 @@ func TestLoadContainerServiceFromFile(t *testing.T) {
 	if containerService.Properties.OrchestratorProfile.OrchestratorVersion != "1.8.12" {
 		t.Errorf("Failed to set orcherstator version when it is set in the json, expected 1.8.12 but got %s", containerService.Properties.OrchestratorProfile.OrchestratorVersion)
 	}
+
+	// Test error scenario
+	containerService, _, err = apiloader.LoadContainerServiceFromFile("../this-file-doesnt-exist.json", true, false, nil)
+	if err == nil {
```

Comment:
```suggestion
	if err != nil {
```

Reply:
No, this test is validating the error scenario.